packages/compass-components/src/components/editor.tsx

```diff
@@ -75,10 +77,21 @@ function Editor({
     ...(!!completer && { enableLiveAutocompletion: true }),
   };
 
+  const editorRef = useRef<AceEditor | null>(null);
+
+  useEffect(() => {
+    if (id && editorRef.current) {
+      // After initial load, assign the id to the text area used by ace.
+      // This is so labels can `htmlFor` the input.
+      editorRef.current.editor.textInput.getElement().id = id;
+    }
+  }, [id]);
```

Comment:
What is the `editorId` that we have below that for then? Do we need two? Should it be one or the other?

Reply:
Ah thanks for pointing out, I'll update the `editorId` variable to be `editorName` as it's not used for the textarea id, and is passed to ReactAce as the `name`.